### ISSUE TITLE
Provide feedback when karma is granted or taken away

### ISSUE THREAD

**Issue by els-pnw** [marlboromoo/willie-karma#4]
Took a stab at this, you may want to verify I have the statements at the appropriate level of the if conditions.

```
diff --git a/karma.py b/karma.py
index 78a08bc..09d20c0 100644
--- a/karma.py
+++ b/karma.py
@@ -156,7 +156,10 @@ def _meet_karma(bot, trigger, parse_fun, karma_fun):
         if who:
             reason = reason if reason else str(None)
             karma_fun(table, who, reason)
+            karma, reason= get_karma(table, who)
+            bot.say("%s: %s, reason: %s" % (who, karma, reason))
 @willie.module.rule('.*\+\+')
 def meet_add_karma(bot, trigger):
     """Update karma status for specify IRC user if get '++' message.
```

**Comment by 0xd3adcafe** [comment]
@eanxgeek, could you open a pull request  ? :+1:

**Comment by 0xd3adcafe** [comment]
Fixed in #6.
